judge/admin/submission.py

```diff
@@ -244,9 +244,10 @@ def language_column(self, obj):
 
     def judge_column(self, obj):
         if obj.is_locked:
-            return format_html('<input type="button" disabled value="Locked"/>')
+            return format_html('<input type="button" disabled value="{0}"/>', _('Locked'))
         else:
-            return format_html('<input type="button" value="Rejudge" onclick="location.href=\'{}/judge/\'" />', obj.id)
+            return format_html('<input type="button" value="{0}" onclick="location.href=\'{1}/judge/\'" />',
```

Comment:
While we're changing this code, could we update this to reverse the URL instead of this hack?

Reply:
Done.